### PR TITLE
Adding Arabic language

### PR DESCRIPTION
**Pull request by mhmdAljefri**

### CODE REVIEW

**Comment by mhmdAljefri** [comment]
hey @mohammed-bahumaish can you check this PR :pray:

File: packages/ui/src/i18n.ts [hunk 1]
```diff
@@ -66,6 +65,48 @@ const dictJa: { [key in keyof DictEn]: string } = {
...
-const i18n = (lang: Lang, key: keyof DictEn) => (lang === DEFAULT_LANG ? dictEn[key] : dictJa[key]);
+const dictAr: { [key in keyof DictEn]: string } = {
+  cancel: 'إلغاء',
+  field: 'الحقل',
+  fieldName: 'اسم الحقل',
+  require: 'مطلوب',
+  uniq: 'يجب أن يكون فريداً',
+  inputExample: 'مثال',
+  edit: 'تعديل',
+  plsSelect: 'اختر',
```
Comment (aalnussairi) [inline packages/ui/src/i18n.ts:76]: يرجى الاختيار

File: packages/ui/src/i18n.ts [hunk 2]
```diff
@@ -66,6 +65,48 @@ const dictJa: { [key in keyof DictEn]: string } = {
...
+  noKeyName: 'لا يوجد اسم للحقل',
+  fieldsList: 'قائمة الحقول',
+  addNewField: 'إضافة حقل جديد',
+  editField: 'تعديل الحقل',
+  type: 'النوع',
+  goToFirst: 'الذهاب إلى الصفحة الأولى',
+  goToPrevious: 'الذهاب إلى الصفحة السابقة',
+  goToNext: 'الذهاب إلى الصفحة التالية',
+  goToEnd: 'الذهاب إلى الصفحة الأخيرة',
+  select: 'اختر',
```
Comment (aalnussairi) [inline packages/ui/src/i18n.ts:90]: اختار

File: packages/ui/src/i18n.ts (hunk 1 above)
Comment (aalnussairi) [reply packages/ui/src/i18n.ts:76]: Or 

الرجاء أختار

But the previous comment is more correct

**Review by aalnussairi — CHANGES_REQUESTED** [review]
minor changes requested, اختر is incorrect. It should be اختار. Otherwise, it's all good.

**Comment by azenyem** [comment]
@mhmdAljefri @hand-dot could you finish the merging process please? I would like to add two more languages...

Fields `plsSelect`, `plsAddNewField`, `goToFirst`, `goToPrevious`, `goToNext`, `goToEnd`, `select`, `zoomIn`, `zoomOut` became obsolete since the pull request.

Thank you

**Comment by hand-dot** [comment]
Sorry for the late reply...
I will merge it since it looks like you have reviewed it!

I will include it in the next release, so please wait a little longer for the release!

**Pull request merged**